<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAboutDialog</class>
 <widget class="QDialog" name="CAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>748</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About swift</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="vl_Dialog">
   <item>
    <widget class="QGroupBox" name="gb_Version">
     <property name="title">
      <string>Version</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Details">
      <item>
       <widget class="QLabel" name="lbl_VersionValue">
        <property name="text">
         <string>Version ....</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Details">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Version">
      <item>
       <widget class="QPlainTextEdit" name="pte_Info">
        <property name="documentTitle">
         <string>about swift details</string>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_swiftLicense">
     <property name="title">
      <string>swift license</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="tbr_swiftLicense"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_ThirdPartyLicenses">
     <property name="title">
      <string>Third-Party licenses</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextBrowser" name="tbr_ThirdPartyLicenses"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_AboutDialog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pte_Info</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bb_AboutDialog</sender>
   <signal>accepted()</signal>
   <receiver>CAboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
